src/include/storage/fd.h

```diff
@@ -42,7 +42,18 @@
 #define FD_H
 
 #include <dirent.h>
+#include "postgres_ext.h"
 
+/*
+ * Structure that is used for temp_tablespaces and temp_spill_files_tablespaces.
+ * When count is -1, this has not been set in the current transaction.
+ */
+typedef struct BackendTempTableSpaces
+{
+	Oid *tempTableSpaces;
```

Comment:
I think that the definition of this structure can be transferred to the fd.c file, and also formatted by pgindent.